lib/src/lib.rs

```diff
@@ -124,15 +126,15 @@ impl Measurement {
 }
 
 pub fn print_duration(title: &str, duration: time::Duration) {
-    println!(
+    debug!(
         "{title}{}.{:03} seconds",
         duration.as_secs(),
         duration.subsec_millis()
     );
 }
 
 pub fn inplace_print(title: &str) {
-    print!("\r{title}");
+    debug!("\r{title}");
```

Comment:
breaks the inplace printing for the nice single line updating of progress though... :(